main.js

```diff
@@ -0,0 +1,134 @@
+var board = new Map([[0, ''], [1, ''], [2, ''], [3, ''], [4, ''], [5, ''], [6, ''], [7, ''], [8, '']]);
+var turn = true;
+let turnCount = 0;
+
+function buildBoard() {
+    document.getElementById('cell-8').innerHTML = board.get(8);
+    document.getElementById('cell-0').innerHTML = board.get(0);
+    document.getElementById('cell-1').innerHTML = board.get(1);
+    document.getElementById('cell-2').innerHTML = board.get(2);
+    document.getElementById('cell-3').innerHTML = board.get(3);
+    document.getElementById('cell-4').innerHTML = board.get(4);
+    document.getElementById('cell-5').innerHTML = board.get(5);
+    document.getElementById('cell-6').innerHTML = board.get(6);
+    document.getElementById('cell-7').innerHTML = board.get(7);
+};
```

Comment:
If you define this function below where you define your cell variables you can just use those variables instead of doing a whole new DOM query.